Contributing/Issue_Priorities.md

```diff
@@ -0,0 +1,32 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## AMP GitHub Issue Priorities
+The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
+These new guidelines can give you also a good overview of when to expect updates or closure of issues.
+
+In order to add a priority to your issues use our [list of GitHub labels](https://github.com/ampproject/amphtml/labels).
+
+### Priorities & Guidelines
+
+
+ Priority           | What it means          | Guidelines for Bugs              | Guidelines  for FRs               
+--------------------| ---------------------- | ---------------------------------| --------------------------------- 
+P0: Drop Everything | <ul><li>Outage</li><li>Critical production issue</li></ul>    | <ul><li>Drop everything until this is fixed</li><li>Should be assigned and accepted within 24 hours of filing</li><li>Provide regular bug update on status and ETA of the fix, once every day after acceptance</li><li>Fixed in 7 days (next release or patch release, whichever is sooner)</li></ul>  |  <ul><li>A FR is almost never a P0</li></ul>
```

Comment:
"almost never" in "A FR is almost never a P0" feels like it leaves some wiggle room that I don't think is there.  I can't imagine a feature request that we'd tell people not to stop working on it until it was done, whereas a P0 may have that expectation.